internal/services/datafactory/data_factory_linked_service_azure_blob_storage_resource.go

```diff
@@ -54,7 +55,20 @@ func resourceDataFactoryLinkedServiceAzureBlobStorage() *pluginsdk.Resource {
 				Optional:     true,
 				Sensitive:    true,
 				ValidateFunc: validation.StringIsNotEmpty,
-				ExactlyOneOf: []string{"connection_string", "sas_uri", "service_endpoint"},
+				ExactlyOneOf: []string{"connection_string", "insecure_connection_string", "sas_uri", "service_endpoint"},
+			},
+
+			"insecure_connection_string": {
```

Comment:
would this be better as
```suggestion
			"connection_string_insecure": {
```